homeassistant/components/litterrobot/strings.json

```diff
@@ -118,6 +118,9 @@
           "spf": "Pinch detect at startup"
         }
       },
+      "total_cycles": {
+        "name": "Total cycles"
```

Comment:
let's add a translatable unit of measurement `cycles`

Reply:
Is there anything more I need to do than just add `"unit_of_measurement": "cycles"`? I looked at the SmartThings integration which had this and didn't see anything else